packages/playground/windows/playground-win32/Playground-win32.vcxproj

```diff
@@ -70,7 +69,6 @@
     <ClCompile>
       <CallingConvention>StdCall</CallingConvention>
       <ConformanceMode>true</ConformanceMode>
-      <LanguageStandard>stdcpp17</LanguageStandard>
       <PrecompiledHeader>Use</PrecompiledHeader>
```

Comment:
this shouldn't be required. default is C++14, we should be able to use 17 too though

Reply:
I was asked to remove this tag from projects, since we set it globally in React.Cpp.props.